<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ferias</class>
 <widget class="QDialog" name="ferias">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>343</width>
    <height>228</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>321</width>
     <height>201</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="sizeConstraint">
     <enum>QLayout::SetFixedSize</enum>
    </property>
    <property name="spacing">
     <number>2</number>
    </property>
    <item row="2" column="1" colspan="2">
     <widget class="QLabel" name="label_nome_2">
      <property name="text">
       <string>Nome</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QDateEdit" name="dateEdit_inicio"/>
    </item>
    <item row="6" column="0">
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>60</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="7" column="2">
     <widget class="QPushButton" name="pushButton_criarferias">
      <property name="text">
       <string>Criar Férias</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_inicio">
      <property name="text">
       <string>Data de início</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_funcionario">
      <property name="text">
       <string>Funcionário</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QSpinBox" name="spinBox_duracao"/>
    </item>
    <item row="4" column="1">
     <widget class="QLabel" name="label_duracao">
      <property name="text">
       <string>Duração</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="label_nome">
      <property name="text">
       <string>Nome</string>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QLabel" name="label_fim">
      <property name="text">
       <string>Data fim</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QComboBox" name="comboBox_funcionario">
      <item>
       <property name="text">
        <string>Funcionário 01</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Funcionário 02</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Funcionário 03</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Funcionário 04</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Funcionário 05</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Funcionário 06</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Funcionário 07</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Funcionário 08</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Funcionário 09</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Funcionário 10</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="5" column="2">
     <widget class="QLabel" name="label_fim_data">
      <property name="text">
       <string>Data fim</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Preferred</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
